6238_prototype/src/main/java/frc/robot/subsystems/DriveSubsystem.java

```diff
@@ -21,8 +25,14 @@
     private final WPI_TalonFX talonRightFollowerTwo = new WPI_TalonFX(Constants.RIGHT_FOLLOWER_ID_TWO);
     private final DifferentialDrive robotDrive = new DifferentialDrive(talonLeftLeader, talonRightLeader);
 
+    private final double kCountsPerRev = 2048;
+    private final double kGearRatio = 20;
+    private final double kWheelRadiusInches = 2;
+
```

Comment:
We should test this code by using driving the robot manually and measuring the distance it goes.